modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderResourceImpl.java

```diff
@@ -534,10 +534,10 @@ private Long[] _getOrderItemIds(OrderItem[] orderItems) {
 	}
 
 	private Page<Order> _getOrdersPage(
-		long companyId, Filter filter, Pagination pagination, String search,
-		Sort[] sorts,
-		UnsafeFunction<Document, Order, Exception> transformUnsafeFunction,
-		boolean useSearchResultPermissionFilter)
+			long companyId, Filter filter, Pagination pagination, String search,
```

Comment:
@RenanUmV This is not a big problem, but we should avoid it. When you merge the helpers, run SF before making each commit, so the merging commits won't contain unnecessary indent changes, and this won't appear in your SF commit which mainly aims to inline methods.